internal/controllers/lvmcluster/status.go

```diff
@@ -69,6 +69,10 @@ func setResourcesAvailableConditionFalse(instance *lvmv1alpha1.LVMCluster, recon
 }
 
 func setVolumeGroupsReadyCondition(instance *lvmv1alpha1.LVMCluster, vgNodeStatusList *lvmv1alpha1.LVMVolumeGroupNodeStatusList) {
+	if len(vgNodeStatusList.Items) < 1 {
```

Comment:
I think this check is too naive. we should check that for every vg in the device class we have all the volume groups in the node status that we would expect based on the node selector. For example now, if no deviceClasses are present, this would be a failed status, but with the Microshift integration this is a valid use case.